groceries.txt

```diff
@@ -9,4 +9,6 @@ corn on the cob
 hamburger buns
 cookies
 pasta sauce
-ice cream
\ No newline at end of file
+ice cream
+soap
```

Comment:
If you get this message, try deleting the `soap` entry, and then commit that change and push the branch **again**.  You should see this PR update with the latest changes to the `soap` branch.